lisa/tools/virtualclient.py

```diff
@@ -248,40 +248,53 @@ def download_raw_data_file(self) -> Path:
         )
         return local_path
 
-    def send_metrics_message(self, metrics_file: Path) -> None:
+    def send_metrics_messages(
+        self,
+        metrics_file: Path,
+        node: "Node",
+        test_result: "TestResult",
+    ) -> None:
         with open(metrics_file, mode="r", newline="") as csvfile:
             reader = csv.DictReader(csvfile)
-            metric_messages: List[VCMetrics] = []
             for row in reader:
-                metric = VCMetrics(
-                    time_stamp=self._parse_timestamp(row["Timestamp"]),
-                    experiment_id=row["ExperimentId"],
-                    client_id=row["ClientId"],
-                    profile=row["Profile"],
-                    profile_name=row["ProfileName"],
-                    tool_name=row["ToolName"],
-                    scenario_name=row["ScenarioName"],
-                    scenario_start_time=self._parse_timestamp(row["ScenarioStartTime"]),
-                    scenario_end_time=self._parse_timestamp(row["ScenarioEndTime"]),
-                    metric_categorization=row["MetricCategorization"],
-                    metric_name=row["MetricName"],
-                    metric_value=Decimal(row["MetricValue"]),
-                    metric_unit=row["MetricUnit"],
-                    metric_description=row["MetricDescription"],
-                    metric_relativity=row["MetricRelativity"],
-                    execution_system=row["ExecutionSystem"],
-                    operating_system_platform=row["OperatingSystemPlatform"],
-                    operation_id=row["OperationId"],
-                    operation_parent_id=row["OperationParentId"],
-                    app_host=row["AppHost"],
-                    app_name=row["AppName"],
-                    app_version=row["AppVersion"],
-                    app_telemetry_version=row["AppTelemetryVersion"],
-                    tags=row["Tags"],
+                fields = {
+                    "time": self._parse_timestamp(row["Timestamp"]),
```

Comment:
time is duplicated with time_stamp

Reply:
I thought to keep vc fields. But I will remove vc fields, if they are defined by us already.